clash-prelude/tests/Test/Tasty/Hedgehog/Extra.hs

```diff
@@ -0,0 +1,20 @@
+{-|
+Convenience functions \"missing\" in "Test.Tasty.Hedgehog"
+
+Copyright  :  (C) 2022, QBayLogic B.V.
+License    :  BSD2 (see the file LICENSE)
+Maintainer :  QBayLogic B.V. <devops@qbaylogic.com>
```

Comment:
Doesn't the module header have to be before the description?

Reply:
Jup